validation/poststart_fail.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	"time"
+
+	tap "github.com/mndrix/tap-go"
+	rspec "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/opencontainers/runtime-tools/specerror"
+	"github.com/opencontainers/runtime-tools/validation/util"
+	uuid "github.com/satori/go.uuid"
+)
+
+func main() {
+	t := tap.New()
+	t.Header(0)
+
+	config := util.LifecycleConfig{
+		Actions: util.LifecycleActionCreate | util.LifecycleActionStart | util.LifecycleActionDelete,
+		PreCreate: func(r *util.Runtime) error {
+			r.SetID(uuid.NewV4().String())
+			g := util.GetDefaultGenerator()
+			poststart := rspec.Hook{
+				Path: fmt.Sprintf("%s/%s/bin/false", r.BundleDir, g.Spec().Root.Path),
```

Comment:
Can we use [`filepath.Join`][1] here?

[1]: https://golang.org/pkg/path/filepath/#Join

Reply:
OK, I'll update it.